server/routes/bookedtoday/summaryWithRecordController.ts

```diff
@@ -9,7 +9,10 @@ export default class SummaryWithRecordController {
       const { id } = req.params
       const arrival = await this.expectedArrivalsService.getArrival(id)
       const singleMatch = arrival.potentialMatches[0]
-      return res.render('pages/bookedtoday/summaryWithRecord.njk', { arrival, singleMatch })
+      const prisonerSummaryDetails = await this.expectedArrivalsService.getPrisonerSummaryDetails(
+        singleMatch.prisonNumber
+      )
+      return res.render('pages/bookedtoday/summaryWithRecord.njk', { arrival, prisonerSummaryDetails })
```

Comment:
Felt it was better to pass both the `arrival` and `prisonerSummaryDetails` through rather than create one object of type 'PrisonerDetails' which would have been made up of the PER and NOMIS data when the 'PrisonerDetails' should be data from just NOMIS.

Reply:
([DCS-2034](https://dsdmoj.atlassian.net/browse/DCS-2034) details the data source (PER/NOMIS) for each field on the summary page)

Reply:
refactored for latest commit to call single service method in controller and return arrival and summary as one object.